data/anaconda.conf

```diff
@@ -138,6 +138,7 @@ selinux = -1
 #
 #   DEFAULT   Choose the type by platform.
 #   EXTLINUX  Use extlinux as the bootloader.
+#   SYSTEMD   Use systemd-boot as the bootloader.
```

Comment:
Definitely +1 here, "systemd" is so confusing. The bootloader is called systemd-boot, so why not use this name?

Reply:
Yah, I guess people could be confused that there is an option to avoid systemd entirely. Lol

No strong opinions here, although I think "sdboot" is probably a better choice than "systemd-boot" for two reasons, one it doesn't have a "-" in the name, which will avoid possible name escaping. To a lesser extent, its less characters I have to type on a serial console when testing :) it.

Reply:
@jlinton agreed that sdboot is a better name.